sources/models/defaults.toml

```diff
@@ -77,6 +77,15 @@ restart-commands = ["/usr/bin/host-containers"]
 [metadata.settings.host-containers]
 affected-services = ["host-containers"]
 
+# Network
+
+[configuration-files.proxy-env]
+path = "/etc/shared/services/network/proxy/env"
+template-path = "/usr/share/templates/proxy-env"
+
+[metadata.settings.network]
+affected-services = ["containerd", "ecs", "kubernetes", "host-containerd"]
```

Comment:
This list strikes me as odd since ecs and kubernetes are mutually exclusive. I guess it doesn't matter since since, presumably, the presence of non-existent services in the list is harmless.

Reply:
I suppose I would expect to override this within the models for `aws-k8s-*` and `aws-ecs`, as you did for `settings.metrics`.